sirius-pipeline/README.md

```diff
@@ -1,2 +1,41 @@
-# Sirius Pipeline P4 Behavior Models
-**TODO**
\ No newline at end of file
+# Sirius Pipeline
+
+## Build the environment
+```
+make docker
+```
+
+## Build pipeline
+```
+make clean
+bmv2/sirius_pipeline.bmv2/sirius_pipeline.json
+```
+
+## Run software switch
+```
+make run_switch
```

Comment:
I get errors:
```
DASH-1/sirius-pipeline$ make run-switch 
docker run -it --name simple_switch -v /home/chris/nvidia/DASH-1/sirius-pipeline/bmv2:/bmv2 -v /home/chris/nvidia/DASH-1/sirius-pipeline/behavioral-model:/bm-src --network=host --rm bmv2-chris simple_switch --log-console --interface 0@veth0 --interface 1@veth2 /bmv2/sirius_pipeline.bmv2/sirius_pipeline.json
Calling target program-options parser
Invalid match type: 'list'
Makefile:21: recipe for target 'run-switch' failed
make: *** [run-switch] Error 1
```

Reply:
This match type is not yet supported by the simulator - I will provide another update to have a version of p4 code that can be compiled into the current BMv2.